it/connectors/SelfEmploymentConnectorISpec.scala

```diff
@@ -237,4 +238,31 @@ class SelfEmploymentConnectorISpec extends WiremockSpec with IntegrationBaseSpec
       result shouldBe ().asRight
     }
   }
+
+  "hasOtherIncomeSources" must {
+    "return true when there is an other income" in {
+      val responseBody = "true"
+      stubGetWithResponseBody(checkForOtherIncomeSourcesUrl, OK, responseBody, headersSentToBE)
+
+      val result = connector.hasOtherIncomeSources(taxYear, nino, mtditid).value.futureValue
+
+      result shouldBe Right(true)
+    }
+
+    "return false on there is an other income" in {
```

Comment:
This description isn't very clear, or different from the 'true' case.
Could replace with "return 'true' from downstream" and "return 'false' from downstream".
But doesn't matter much, up to you